Lib/imghdr.py

```diff
@@ -0,0 +1,170 @@
+"""Recognize image file formats based on their first few bytes."""
+
+#from os import PathLike
```

Comment:
Generally our method for modifying stdlib modules is to add the unmodified module in one commit, and then modify what you need to in a separate commit, preferably marking the modification with `# XXX <description and reason for modification>`